Started UtBot Contest, classfileDir = /var/benchmarks/projects/guava/guava/target/classes, classpathString=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, outputDir=/home/utbot_mocks/temp/testcases, mocks=Other classes: Mockito
22:39:06.152 | INFO  | Started: warmup: kotlin reflection :: init
22:39:06.232 | INFO  | Finished (in 0.079 sec): warmup: kotlin reflection :: init 
[INIT]
>> [RUN] com.google.common.base.Throwables 120
22:39:06.242 | DEBUG | -----------------------------------------------------------------------------
22:39:06.242 | INFO  | Contest.runGeneration: Time budget: 120000 ms, Generation timeout=102000 ms, classpath=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, methodNameFilter=null
22:39:06.244 | INFO  | ClassUnderTest[ FQN: com.google.common.base.Throwables
    classfileDir: /var/benchmarks/projects/guava/guava/target/classes
    testClassSimpleName: ThrowablesTest
    generatedTestFile: /home/utbot_mocks/temp/testcases/com/google/common/base/ThrowablesTest.java
    generatedTestsSourcesDir: /home/utbot_mocks/temp/testcases
]
22:39:06.266 | INFO  | Started: class com.google.common.base.Throwables
22:39:06.266 | INFO  | Started: preparation class class com.google.common.base.Throwables: kotlin reflection :: run
22:39:06.297 | INFO  | Finished (in 0.030 sec): preparation class class com.google.common.base.Throwables: kotlin reflection :: run 
22:39:06.298 | INFO  | Started: 2nd optional soot initialization
22:39:06.299 | INFO  | Finished (in 0.001 sec): 2nd optional soot initialization 
22:39:06.307 | DEBUG |  ... 
22:39:06.317 | INFO  | Started: method com.google.common.base.Throwables.getRootCause(Throwable)
22:39:06.318 | INFO  |  -- Remaining time budget: 101934 ms, #remaining_methods: 20, budget for method: 5096 ms, solver timeout: 1000 ms, budget for symbolic execution: 3896 ms, budget for concrete execution: 1200 ms,  -- 
22:39:06.651 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)1
22:39:07.009 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)2
22:39:07.303 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)3
22:39:07.961 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)4
22:39:08.462 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)5
22:39:08.883 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)6
22:39:10.211 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getRootCause(Throwable)
22:39:11.412 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getRootCause(Throwable)
22:39:11.516 | INFO  | Finished (in 5.198 sec): method com.google.common.base.Throwables.getRootCause(Throwable) 
<StatsForMethod> :
	#generatedTC=6
	WITH NO EXCEPTIONS
22:39:11.519 | DEBUG |  ... 
22:39:11.520 | INFO  | Started: method com.google.common.base.Throwables.propagate(Throwable)
22:39:11.520 | INFO  |  -- Remaining time budget: 96723 ms, #remaining_methods: 19, budget for method: 5090 ms, solver timeout: 1000 ms, budget for symbolic execution: 3890 ms, budget for concrete execution: 1200 ms,  -- 
22:39:11.536 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)1
22:39:12.207 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)2
22:39:12.370 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)3
22:39:12.544 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)4
22:39:12.545 | INFO  | Finished (in 1.024 sec): method com.google.common.base.Throwables.propagate(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
22:39:12.570 | DEBUG |  ... 
22:39:12.571 | INFO  | Started: method com.google.common.base.Throwables.throwIfUnchecked(Throwable)
22:39:12.571 | INFO  |  -- Remaining time budget: 95672 ms, #remaining_methods: 18, budget for method: 5315 ms, solver timeout: 1000 ms, budget for symbolic execution: 4115 ms, budget for concrete execution: 1200 ms,  -- 
22:39:12.579 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)1
22:39:12.834 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)2
22:39:12.987 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)3
22:39:13.145 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)4
22:39:13.145 | INFO  | Finished (in 0.574 sec): method com.google.common.base.Throwables.throwIfUnchecked(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
22:39:13.148 | DEBUG |  ... 
22:39:13.148 | INFO  | Started: method com.google.common.base.Throwables.getCausalChain(Throwable)
22:39:13.149 | INFO  |  -- Remaining time budget: 95095 ms, #remaining_methods: 17, budget for method: 5593 ms, solver timeout: 1000 ms, budget for symbolic execution: 4393 ms, budget for concrete execution: 1200 ms,  -- 
22:39:13.159 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)1
22:39:14.272 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)2
22:39:17.542 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getCausalChain(Throwable)
22:39:18.126 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)3
22:39:18.743 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getCausalChain(Throwable)
22:39:18.769 | INFO  | Finished (in 5.620 sec): method com.google.common.base.Throwables.getCausalChain(Throwable) 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
22:39:18.916 | DEBUG |  ... 
22:39:18.916 | INFO  | Started: method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)
22:39:18.917 | INFO  |  -- Remaining time budget: 89327 ms, #remaining_methods: 16, budget for method: 5582 ms, solver timeout: 1000 ms, budget for symbolic execution: 4382 ms, budget for concrete execution: 1200 ms,  -- 
22:39:18.929 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
22:39:19.298 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)1
22:39:19.433 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)2
22:39:19.933 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)3
22:39:19.980 | INFO  | Finished (in 1.063 sec): method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=3
	FAILED 1 time(s) with 1 different exception(s)"
22:39:19.991 | DEBUG |  ... 
22:39:19.992 | INFO  | Started: method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)
22:39:19.993 | INFO  |  -- Remaining time budget: 88251 ms, #remaining_methods: 15, budget for method: 5883 ms, solver timeout: 1000 ms, budget for symbolic execution: 4683 ms, budget for concrete execution: 1200 ms,  -- 
22:39:19.998 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
22:39:20.045 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)1
22:39:20.814 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)2
22:39:21.145 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)3
22:39:21.327 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)4
22:39:21.328 | INFO  | Finished (in 1.335 sec): method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=4
	FAILED 1 time(s) with 1 different exception(s)"
22:39:21.360 | DEBUG |  ... 
22:39:21.361 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable)
22:39:21.361 | INFO  |  -- Remaining time budget: 86882 ms, #remaining_methods: 14, budget for method: 6205 ms, solver timeout: 1000 ms, budget for symbolic execution: 5005 ms, budget for concrete execution: 1200 ms,  -- 
22:39:21.368 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)1
22:39:21.485 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)2
22:39:22.054 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)3
22:39:22.189 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)4
22:39:22.374 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)5
22:39:22.375 | INFO  | Finished (in 1.014 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable) 
<StatsForMethod> :
	#generatedTC=5
	WITH NO EXCEPTIONS
22:39:22.399 | DEBUG |  ... 
22:39:22.400 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class)
22:39:22.400 | INFO  |  -- Remaining time budget: 85843 ms, #remaining_methods: 13, budget for method: 6603 ms, solver timeout: 1000 ms, budget for symbolic execution: 5403 ms, budget for concrete execution: 1200 ms,  -- 
22:39:22.405 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
22:39:23.796 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)1
22:39:23.818 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)2
22:39:24.223 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)3
22:39:24.374 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)4
22:39:24.562 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)5
22:39:24.765 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)6
22:39:24.766 | INFO  | Finished (in 2.365 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=6
	FAILED 1 time(s) with 1 different exception(s)"
22:39:24.828 | DEBUG |  ... 
22:39:24.829 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)
22:39:24.829 | INFO  |  -- Remaining time budget: 83414 ms, #remaining_methods: 12, budget for method: 6951 ms, solver timeout: 1000 ms, budget for symbolic execution: 5751 ms, budget for concrete execution: 1200 ms,  -- 
22:39:24.834 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
22:39:26.168 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)1
22:39:26.702 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)2
22:39:28.119 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)3
22:39:28.411 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)4
22:39:28.412 | INFO  | Finished (in 3.582 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class) 
<StatsForMethod> :
	#generatedTC=4
	FAILED 1 time(s) with 1 different exception(s)"
22:39:28.536 | DEBUG |  ... 
22:39:28.536 | INFO  | Started: method com.google.common.base.Throwables.getCauseAs(Throwable, Class)
22:39:28.537 | INFO  |  -- Remaining time budget: 79707 ms, #remaining_methods: 11, budget for method: 7246 ms, solver timeout: 1000 ms, budget for symbolic execution: 6046 ms, budget for concrete execution: 1200 ms,  -- 
22:39:28.542 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
22:39:28.645 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)1
22:39:29.067 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)2
22:39:29.161 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)3
22:39:29.712 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)4
22:39:30.163 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)5
22:39:30.725 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)6
22:39:31.184 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)7
22:39:31.434 | INFO  | Finished (in 2.897 sec): method com.google.common.base.Throwables.getCauseAs(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=7
	FAILED 1 time(s) with 1 different exception(s)"
22:39:31.543 | DEBUG |  ... 
22:39:31.544 | INFO  | Started: method com.google.common.base.Throwables.getStackTraceAsString(Throwable)
22:39:31.544 | INFO  |  -- Remaining time budget: 76699 ms, #remaining_methods: 10, budget for method: 7669 ms, solver timeout: 1000 ms, budget for symbolic execution: 6469 ms, budget for concrete execution: 1200 ms,  -- 
22:39:31.555 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)1
22:39:32.310 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)2
22:39:32.311 | INFO  | Finished (in 0.767 sec): method com.google.common.base.Throwables.getStackTraceAsString(Throwable) 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
22:39:32.354 | DEBUG |  ... 
22:39:32.355 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTrace(Throwable)
22:39:32.355 | INFO  |  -- Remaining time budget: 75889 ms, #remaining_methods: 9, budget for method: 8432 ms, solver timeout: 1000 ms, budget for symbolic execution: 7232 ms, budget for concrete execution: 1200 ms,  -- 
22:39:39.587 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTrace(Throwable)
22:39:40.788 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.lazyStackTrace(Throwable)
22:39:40.790 | INFO  | Finished (in 8.435 sec): method com.google.common.base.Throwables.lazyStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
22:39:41.258 | DEBUG |  ... 
22:39:41.258 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTraceIsLazy()
22:39:41.258 | INFO  |  -- Remaining time budget: 66985 ms, #remaining_methods: 8, budget for method: 8373 ms, solver timeout: 1000 ms, budget for symbolic execution: 7173 ms, budget for concrete execution: 1200 ms,  -- 
22:39:41.262 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.lazyStackTraceIsLazy()1
22:39:48.434 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTraceIsLazy()
22:39:48.519 | INFO  | Finished (in 7.260 sec): method com.google.common.base.Throwables.lazyStackTraceIsLazy() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
22:39:48.882 | DEBUG |  ... 
22:39:48.883 | INFO  | Started: method com.google.common.base.Throwables.jlaStackTrace(Throwable)
22:39:48.883 | INFO  |  -- Remaining time budget: 59360 ms, #remaining_methods: 7, budget for method: 8480 ms, solver timeout: 1000 ms, budget for symbolic execution: 7280 ms, budget for concrete execution: 1200 ms,  -- 
22:39:49.210 | INFO  | Finished (in 0.327 sec): method com.google.common.base.Throwables.jlaStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
22:39:49.221 | DEBUG |  ... 
22:39:49.222 | INFO  | Started: method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
22:39:49.222 | INFO  |  -- Remaining time budget: 59021 ms, #remaining_methods: 6, budget for method: 9836 ms, solver timeout: 1000 ms, budget for symbolic execution: 8636 ms, budget for concrete execution: 1200 ms,  -- 
22:39:49.238 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ReadingFromKryoException: Failure reading from kryo
	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:392) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: com.esotericsoftware.kryo.KryoException: Encountered unregistered class ID: 30
	at com.esotericsoftware.kryo.util.DefaultClassResolver.readClass(DefaultClassResolver.java:159) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClass(Kryo.java:758) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClassAndObject(Kryo.java:869) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.util.KryoHelper.readCommand(KryoHelper.kt:46) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1$1.invokeSuspend(ConcreteExecutor.kt:256) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:84) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:253) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:101) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30) ~[utbot-junit-contest-1.0.jar:?]
22:39:49.687 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])1
22:39:57.858 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
22:39:59.059 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
22:39:59.451 | INFO  | Finished (in 10.229 sec): method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[]) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
22:39:59.469 | DEBUG |  ... 
22:39:59.470 | INFO  | Started: method com.google.common.base.Throwables.getJLA()
22:39:59.470 | INFO  |  -- Remaining time budget: 48773 ms, #remaining_methods: 5, budget for method: 9754 ms, solver timeout: 1000 ms, budget for symbolic execution: 8554 ms, budget for concrete execution: 1200 ms,  -- 
22:40:07.276 | INFO  | Finished (in 7.805 sec): method com.google.common.base.Throwables.getJLA() 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
22:40:07.694 | DEBUG |  ... 
22:40:07.695 | INFO  | Started: method com.google.common.base.Throwables.getGetMethod()
22:40:07.695 | INFO  |  -- Remaining time budget: 40548 ms, #remaining_methods: 4, budget for method: 10137 ms, solver timeout: 1000 ms, budget for symbolic execution: 8937 ms, budget for concrete execution: 1200 ms,  -- 
22:40:07.728 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getGetMethod()1
22:40:15.312 | INFO  | Finished (in 7.617 sec): method com.google.common.base.Throwables.getGetMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
22:40:15.684 | DEBUG |  ... 
22:40:15.685 | INFO  | Started: method com.google.common.base.Throwables.getSizeMethod()
22:40:15.685 | INFO  |  -- Remaining time budget: 32558 ms, #remaining_methods: 3, budget for method: 10852 ms, solver timeout: 1000 ms, budget for symbolic execution: 9652 ms, budget for concrete execution: 1200 ms,  -- 
22:40:15.693 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getSizeMethod()1
22:40:23.258 | INFO  | Finished (in 7.573 sec): method com.google.common.base.Throwables.getSizeMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
22:40:23.464 | DEBUG |  ... 
22:40:23.465 | INFO  | Started: method com.google.common.base.Throwables.getJlaMethod(String, Class)
22:40:23.465 | INFO  |  -- Remaining time budget: 24779 ms, #remaining_methods: 2, budget for method: 12389 ms, solver timeout: 1000 ms, budget for symbolic execution: 11189 ms, budget for concrete execution: 1200 ms,  -- 
22:40:23.495 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getJlaMethod(String, Class)1
22:40:31.382 | INFO  | Finished (in 7.917 sec): method com.google.common.base.Throwables.getJlaMethod(String, Class) 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
22:40:31.403 | DEBUG |  ... 
22:40:31.403 | INFO  | Started: method com.google.common.base.Throwables.<init>()
22:40:31.404 | INFO  |  -- Remaining time budget: 16840 ms, #remaining_methods: 1, budget for method: 16840 ms, solver timeout: 1000 ms, budget for symbolic execution: 15640 ms, budget for concrete execution: 1200 ms,  -- 
22:40:31.409 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()1
22:40:31.431 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()2
22:40:31.431 | INFO  | Finished (in 0.027 sec): method com.google.common.base.Throwables.<init>() 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
22:40:31.437 | INFO  | Started: Flushing tests for [Throwables] on disk
22:40:31.762 | INFO  | File size for ThrowablesTest: 49 KB
22:40:31.764 | INFO  | Finished (in 0.326 sec): Flushing tests for [Throwables] on disk 
22:40:31.765 | INFO  | Finished (in 85.498 sec): class com.google.common.base.Throwables 
<StatsForClass> :
	canceled by timeout = false
	#methods = 20, 
	#methods started symbolic exploration = 20
	#methods with at least one TC = 17
	#methods with exceptions = 6
	#generated TC = 55
	#coverage = null
[READY]
